common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -219,6 +219,8 @@ public void merge(TachyonConf alternateConf) {
 
   // TODO(binfan): this method should be hidden and only used during initialization and tests.
   public void set(String key, String value) {
+    Preconditions.checkArgument(key != null && value != null,
+        String.format("the key value pair (%s, %s) cannot have null value", key, value));
```

Comment:
Neither the key nor the value can be null. Removed the second value to make it clear.